### ISSUE TITLE
result of paper

### ISSUE THREAD

**Issue by LYF14020510036** [weihua916/powerful-gnns#8]
I am a beginner. I want to ask you how you got the result of your paper.The results of each validation are the max, and then the max in 10?

**Comment by veophi** [comment]
I'm also very curious. I try to run the code and to adjust the hyper-parameters, but I cannot got expected results.

**Comment by xptree** [comment]
Hey, @LYF14020510036 and @veophi  have you solved the problem.

**Comment by weihua916** [comment]
Thank you for your interest! 

To get the results in the paper, you will need to consider tuning hyper-parameters mentioned in the paper (including using --degree_as_tag for COLLAB, IMDB).

Also, in our paper, we did not use validation set (as the dataset sizes are extremely small). Instead, we first took average of validation curves across 10 folds, and then selected a single epoch that achieved the maximum averaged validation accuracy.

Besides, I strongly recommend using graph deep learning libraries that have been recently developed, such as Pytorch Geometric (https://pytorch-geometric.readthedocs.io/en/latest/) and DGL (https://www.dgl.ai/). They provide faster implementation of our GIN algorithm, and are very easy to use.
